functions-methods.js

```diff
@@ -33,4 +71,36 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(arr) {
+    let mailArray = Array.from(arr);
+    let counter = 0;
+    let emailValid = false;
+    let domain
+    for (let i = 0; i < mailArray.length; i++) {
+        if (mailArray[i] !== "@") {
+            counter++;
+            emailValid = false;
+        } else {
+            domain = arr.slice(counter + 1);
+            emailValid = true;
+            break
+        }
+    }
+    let domArray = Array.from(domain);
+    for (let i = 0; i < domArray.length; i++) {
+        if (domArray[domArray.length - 1] === ".") {
+            emailValid = false;
+        } else if (domArray[domArray.length - 1] === ",") {
+            emailValid = false;
+        } else {
+            emailValid = true;
+        }
+
+    }
```

Comment:
Ik snap de for-loop niet helemaal, omdat je de `i` helemaal niet gebruikt. Volgens mij kun je dus zonder
```suggestion
    if (domArray[domArray.length - 1] === ".") {
        emailValid = false;
    } else if (domArray[domArray.length - 1] === ",") {
        emailValid = false;
    } else {
        emailValid = true;
    }
```
```